build/docker/zap-baseline.py

```diff
@@ -314,7 +314,8 @@ def main(argv):
                 time.sleep(5)
                 logging.debug('Delay passive scan check ' + str(delay - (datetime.now() - start_scan).seconds) + ' seconds')
 
-        zap_wait_for_passive_scan(zap)
+        # Allow twice as long as the time taken to spider for the passive scanning
+        zap_wait_for_passive_scan(zap, mins * 60 * 2)
```

Comment:
Other scripts would benefit from this check (e.g. #3531, but if this is going to be added to core then it might not be necessary).

The timeout could also be dependent on the number of messages to scan?